really-unique-id.js

```diff
@@ -33,20 +36,7 @@ function hammerTime () {
    */
   var doomsDay = new Date()
   doomsDay.setFullYear('2239')
-  return format((new Date()).getTime(), doomsDay.getTime())
-}
-
-/**
- * Returns an 32 bit integer based on a finger print
- * produced by concatinating strings that are usually
- * unique to each user.
- *
- * @returns {Number} fingerPrintId
- */
-function fingerPrintId () {
-  var maxVal = Math.pow(2, 32) - 1
-  var fingerPrint = navigator.userAgent + document.cookie
-  return format(hash(fingerPrint), maxVal)
+  return format((new Date()).getTime() + incrementor++, doomsDay.getTime())
```

Comment:
not sure if doomsday adds any uniqueness? Like its a fixed number of ms from now, but new Date().getTime() is also a fixed number of ms from now? So they're like the same number, but with a slightly different offset